Lib/typing.py

```diff
@@ -2412,10 +2414,72 @@ def utf8(value: bytes) -> bytes: ...
       def utf8(value: str) -> bytes: ...
       def utf8(value):
           # implementation goes here
+
+    Overload definition can be retrieved at runtime using the
```

Comment:
```suggestion
    The list of overloads for a function can be retrieved at runtime using the
```

Reply:
Thanks, I should have noticed that. I've been careful to avoid promising that it actually returns a list though:
```suggestion
    The overloads for a function can be retrieved at runtime using the
```